src/utils/socket_listener.cxx

```diff
@@ -51,34 +51,27 @@
 #include <signal.h>
 #include <unistd.h>
 
-#include "utils/socket_listener.hxx"
-
-#include "utils/macros.h"
+#include "nmranet_config.h"
 #include "utils/logging.h"
+#include "utils/macros.h"
+#include "utils/socket_listener.hxx"
```

Comment:
please do keep the `$self.hxx` at the very top of the include list. Having it in the middle/bottom often hides missing includes.
In fact it should be above the `#include <...>` as well

Reply:
Rearranged.